network_modeling/configs/templates/1.3/aruba/sw-edge.primary.j2

```diff
@@ -0,0 +1,155 @@
+{#- edge_router #}
+{%- include '1.3/aruba/common/unknown.j2' %}
+hostname sw-edge-001
+{%- include '1.3/aruba/common/banner-motd.j2' %}
+no ip icmp redirect
+vrf Customer
+vrf keepalive
+ntp server {{ variables.NCN_W001 }}
+ntp server {{ variables.NCN_W002 }}
+ntp server {{ variables.NCN_W003 }}
+ntp enable
+
+ssh server vrf default
+ssh server vrf keepalive
+ssh server vrf mgmt
+ssh server vrf Customer
+
+interface mgmt
+    no shutdown
+    ip static 172.30.52.119/20
+
+vlan {{ variables.NATIVE_VLAN }}
+vlan {{ variables.CHN_VLAN }}
+    name CHN
+
+
+interface lag 256
+    no shutdown
+    description ISL link
+    no routing
+    vlan trunk native 1 tag
+    vlan trunk allowed all
+    lacp mode active
+interface lag 1 multi-chassis
+    no shutdown
+    description sw-spine
+    no routing
+    vlan trunk native 1
+    vlan trunk allowed all
+    lacp mode active
+    lacp fallback
+    spanning-tree port-type admin-edge
+interface 1/1/1
```

Comment:
assuming that these are just hardcoded for now?

Reply:
Per feedback from luke. Deleting port configs for now. I think we should stand up a better port strategy for HSN connections (and whatever connects to edge), I.e. start reading in pt to pt tab in shcd and create jinja templates for all of these connections. Surely, we can cover these with custom configs but that approach is just prone to trial and error.